django/db/models/query_utils.py

```diff
@@ -27,6 +27,26 @@ class InvalidQuery(Exception):
     pass
 
 
+def merge_dicts(dicts):
+    merged = {}
+
+    # merge in reverse to preference the order of the original list. e.g.,
+    # merge_dicts([a, b]) will preference the keys in a over those in b.
+    for d in reversed(dicts):
+        merged.update(d)
+
```

Comment:
Remove this and the previous blank line to compress it all. Same goes for the subclasses method below. This is just a matter of taste and is entirely subjective.